src/main/java/com/mineboundteam/minebound/registry/BlockRegistry.java

```diff
@@ -22,6 +25,7 @@
 public class BlockRegistry {
     public static final DeferredRegister<Block> BLOCKS = DeferredRegister.create(ForgeRegistries.BLOCKS, MineBound.MOD_ID);
 
+    public static final RegistryObject<Block> GILDED_DIAMOND_BLOCK = BLOCKS.register("gilded_diamond_block", () -> new Block(Properties.copy(Blocks.OBSIDIAN).sound(SoundType.METAL)));
```

Comment:
Make this copy gold or diamond block and set the hardness up. We don't want it to take as long as obsidian to break